pages/dmp_supporting_info/support_04_store_protect.md

```diff
@@ -58,52 +58,53 @@ Alternatively, you can add storage guidelines manually. In collaboration project
 ### Expected total volume of data in the project
 You are advised to roughly estimate the total data volume as early as possible in order to budget for connected costs. You should also take into account that you might produce intermediate data that also requires space, even if some of these data may be discarded in the end of the project. It may be worth to consider if all data needs to be accessible all the time ('hot' storage) or whether using 'cold' storage could be considered for parts of the data. Deciding early on which data needs to be preserved long-term and timely archiving is also advisable to save storage resources.
 
```

Comment:
```suggestion
In case you are planning computationally demanding data analysis, see also the compute power planning in the chapter [Processing and analysing data.](/pages/support_07_process_analyse)
```

Reply:
Still there, moved down.